docs/en-US/component/upload.md

```diff
@@ -89,53 +89,106 @@ upload/manual
+| on-success                    | hook function when uploaded successfully.                                                                                                                                             | ^[Function]`(response: any, uploadFile: UploadFile, uploadFiles: UploadFiles) => void`                   | () => void                                                                                                         |
+| on-error                      | hook function when some errors occurs.                                                                                                                                                | ^[Function]`(error: Error, uploadFile: UploadFile, uploadFiles: UploadFiles) => void`                    | () => void                                                                                                         |
+| on-progress                   | hook function when some progress occurs.                                                                                                                                              | ^[Function]`(evt: UploadProgressEvent, uploadFile: UploadFile, uploadFiles: UploadFiles) => void`        | () => void                                                                                                         |
+| on-change                     | hook function when select file or upload file success or upload file fail.                                                                                                            | ^[Function]`(uploadFile: UploadFile, uploadFiles: UploadFiles) => void`                                  | () => void                                                                                                         |
+| on-exceed                     | hook function when limit is exceeded.                                                                                                                                                 | ^[Function]`(files: File[], uploadFiles: UploadUserFile[]) => void`                                      | () => void                                                                                                         |
+| before-upload                 | hook function before uploading with the file to be uploaded as its parameter. If `false` is returned or a `Promise` is returned and then is rejected, uploading will be aborted.      | ^[Function]`(rawFile: UploadRawFile) => Awaitable<void \| undefined \| null \| boolean \| File \| Blob>` | () => void                                                                                                         |
```

Comment:
Perhaps it can be added that it will only be triggered when `auto-upload` is set to `true`. See [comments](https://github.com/element-plus/element-plus/issues/13857#issuecomment-1663992965).

Reply:
updated.